translate/storage/test_properties.py

```diff
@@ -407,3 +407,12 @@ def test_joomla(self):
         assert propunit.name == u'VALUE'
         assert propunit.source == u'I am a "value"'
         assert bytes(propfile) == propsource
+
+    def test_serialize_missing_value(self):
+        propsource = 'key\n'.encode('utf-8')
```

Comment:
When I read the PR description I was expecting the "no value" to mean that it was like `key=` but not like `key`. Please add a test for that case too.